src/gql-loader/register.ts

```diff
@@ -0,0 +1,209 @@
+// Code based on Webpack Loader from https://github.com/apollographql/graphql-tag
+// The MIT License (MIT)
+// Copyright (c) 2020 Meteor Development Group, Inc.
+// Changes Copyright (c) 2020 Mikael Brevik in https://github.com/mikaelbr/graphql-node-import
```

Comment:
I've been spending some time lately having fun with this concept, this is just copypasted from a different repo for this purpose. When it is ready I'll refer to the package instead. So this file is temporary.